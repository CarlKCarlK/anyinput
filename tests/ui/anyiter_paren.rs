use anyinput::anyinput;

#[anyinput]
pub fn any_str_len(s: AnyIter(AnyString)) -> usize {
    s.len()
}

fn main() {}
