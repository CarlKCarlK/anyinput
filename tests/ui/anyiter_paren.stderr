error: Expected <..> generic parameter.
 --> tests/ui/anyiter_paren.rs:4:23
  |
4 | pub fn any_str_len(s: AnyIter(AnyString)) -> usize {
  |                       ^^^^^^^^^^^^^^^^^^
